src/LargeXlsx/Worksheet.cs

```diff
@@ -271,6 +273,15 @@ public void SetSheetProtection(XlsxSheetProtection sheetProtection)
             _sheetProtection = sheetProtection;
         }
 
+        public void SetHeaderFooter(XlsxHeaderFooter headerFooter)
+        {
+            if (!headerFooter.DifferentOddEven && headerFooter.OddHeader == null)
+                throw new ArgumentException("When different header for odd and even pages is not used OddHeader value must be set");
+            if (!headerFooter.DifferentOddEven && headerFooter.OddFooter == null)
```

Comment:
This appears to be a duplicate check. Did you mean to do that single check, or to make a second, different one?

Reply:
Not a duplicate, first if checks `headerFooter.OddHeader == null` and second `headerFooter.OddFooter == null`.